src/main/resources/templates/register.html

```diff
@@ -1,81 +1,118 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html" lang="en">
```

Comment:
Ich bin verwirrt...im PR heißt dieser PR "error page", jetzt änderst du aber auch was bei register und machst Sachen von Diana rückgängig. Das sollte in einem seperaten PR passieren soweit ich weiß

Reply:
done das war gestern zum testen was der unterschied der Seiten ist